app/model/SubscriptionOps.scala

```diff
@@ -54,6 +54,7 @@ object SubscriptionOps extends LazyLogging {
       }
     }
 
+    def firstPrice = firstPlan.charges.price.getPrice(currency).map(_.amount).getOrElse(0F)
```

Comment:
As you're adding this to a string in the JSON in thankyou.scala.html, I think you can do .map(_.pretty).mkString